distribution/ddf-common/src/main/resources-filtered/etc/custom.system.properties

```diff
@@ -132,10 +132,10 @@ org.codice.ddf.system.threadPoolSize=128
 # java.security.manager=net.sourceforge.prograde.sm.PolicyFileGeneratorJSM
 # proGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity
 
-policy.provider=net.sourceforge.prograde.policy.ProGradePolicy
-java.security.manager=net.sourceforge.prograde.sm.ProGradeJSM
-java.security.policy==${karaf.etc}/all.policy
-proGrade.getPermissions.override=sun.rmi.server.LoaderHandler:loadClass,org.apache.jasper.compiler.JspRuntimeContext:initSecurity
+# policy.provider=net.sourceforge.prograde.policy.ProGradePolicy
```

Comment:
❓ I assume these will be handled in the next PR?

Reply:
The developers that did the majority of the maintenance of the security policy are no longer active on the project. The current policy will likely stay disabled until someone that requires it steps up to fix and maintain it.

Reply:
Another issue is that we are using pro-grade which has not been maintained since August 8, 2016. We would first need to switch to using a maintained policy provider that works with Equinox. 

After doing that, we could create a new policy that is inverted from a deny by default to an grant by default policy. Then the policy would focus on known sensitive files in our running system to protect.